ignite/pkg/cosmosclient/cosmosclient.go

```diff
@@ -390,6 +390,74 @@ func (c Client) BroadcastTxWithProvision(accountName string, msgs ...sdktypes.Ms
 	}, nil
 }
 
+// GetBlockTXs returns the transactions in a block.
+// The list of transactions can be empty if there are no transactions
+// in the block at the moment this method is called.
+func (c Client) GetBlockTXs(ctx context.Context, height int64) (txs []TX, err error) {
+	r, err := c.RPC.Block(ctx, &height)
+	if err != nil {
+		return nil, fmt.Errorf("failed to fetch block %d: %w", height, err)
+	}
+
+	blockTime := r.Block.Time.UTC().Format(time.RFC3339Nano)
+	query := fmt.Sprintf("tx.height=%d", height)
+	perPage := 30
```

Comment:
Maybe these could be defined as constants in the cosmosclient scope?

Reply:
The `perPage` is an argument that the [TxSearch](https://pkg.go.dev/github.com/tendermint/tendermint/rpc/client/http#BatchHTTP.TxSearch) method receives as an integer pointer, so it could be a constant but to get the address I should define a variable which would leave the code as:
```go
const defaultTXsPerPage = 30

// ...

perPage := defaultTXsPerPage
c.RPC.TxSearch(ctx, query, false, &page, &perPage, "asc")
```

Otherwise I could use a variable which is fine but semantically is better to use a constant:
```go
var defaultTXsPerPage = 30
```

@ilgooz which one do you favor ?

Reply:
Hey didn't know that. It's up to you. 👍